store/index.js

```diff
@@ -248,9 +274,50 @@ export const actions = {
     account = normalizeAddr(account);
     if (!state.activeAccount) commit('setAccount', account);
     if (state.accounts[account] === true) return; // Already added
-    state.accounts[account] = true;
+    this._vm.$set(state.accounts, account, true);
     commit('addAccount', account);
-  }
+  },
+
+  async detectHalfWrapped({ state, commit }, { ketherContract, nftContract, numBlocks }) {
+    // FIXME: Probably should move this into Wrap or a ResumeWrap component, no need to pollute global state
```

Comment:
@mveytsman I was hoping to move this out of the store and into its own component, but looks like your changes double-down on it being in-store. Do you have feels on this or just a coincidence?

Reply:
As discussed I think we should move it into a component eventually but doing it this way lets us ship now.